student-forms/Kick-Off-2479/bootstrap-setup/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "bootstrap-setup",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "start": "vite",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/bootstrap": "^5.2.6",
+    "sass": "^1.60.0",
```

Comment:
# Correct sass location in the file
Sass suppose to be in dependence not in devdependence